qutebrowser/completion/models/base.py

```diff
@@ -44,6 +44,17 @@ class BaseCompletionModel(QStandardItemModel):
         COLUMN_WIDTHS: The width percentages of the columns used in the
                         completion view.
         DUMB_SORT: the dumb sorting used by the model
+
+    Attributes:
+        filtered_out_cache:
```

Comment:
Would it make more sense for `filtered_out_cache` to be an attribute on `SortFilter` rather than the completion model?